weblab/experiments/tests/test_views.py

```diff
@@ -520,41 +520,42 @@ def test_submatrix_with_too_many_model_ids(self, client, helpers, quick_experime
 
     def test_submatrix_with_protocol_versions(self, client, helpers, quick_experiment_version):
         exp = quick_experiment_version.experiment
-        v1 = str(exp.protocol_version)
-        v2 = str(helpers.add_fake_version(exp.protocol).sha)
+        v1 = exp.protocol_version
+        v2 = helpers.add_fake_version(exp.protocol)
         helpers.add_fake_version(exp.protocol)  # v3, not used
 
         exp2 = make_experiment(
-            exp.model, exp.model_version,
+            exp.model, exp.model.repocache.get_version(exp.model_version.sha),
```

Comment:
I think you can use `exp.model_version` directly here?

Reply:
good spot, thanks